### DISCUSSION TITLE
Foreign keys as primary values [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1891] [Discord Threads]
> 🙋‍♂️ **Harbard**  
#### ```  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/963488073625255997/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### The idea of foreign key is mostly for machines 🙂 and hence we are avoiding that to be set as primary values  
#  
> 💁 **o1lab**  
#### the idea of primary value is that it has to be humanly understandabale  
#  
> 🙋‍♂️ **Harbard**  
#### I understand your point. The thing is, I really need a relational DB for my use case, and I also need non-dev people to work on the DB. So nocoDB seems to be the perfect tool here !   
#### Is my DB structure exotic ? Because I feel like it's pretty commun. Let's say I want to creat a new classroom and attach it to a school, the user will be faced with PK instead of the label.  
![image0](https://cdn.discordapp.com/attachments/963488073566539796/963709834568544277/unknown.png)  
#  
> 🙋‍♂️ **Harbard**  
#### Anyway, I'd be glad to discuss this topic with anyone in the team and try to help ! 🙏  
#  
> 🙋‍♂️ **Harbard**  
#### What is the best place to talk about this ? Here, an issue or GitHub discussions ?  
#  
> 🙋‍♂️ **Harbard**  
#### <a:kbInklingWave:630956774727483422> What do you guys think about that ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963488073566539796)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
